### ISSUE TITLE
Geo Filters (bounding box, distance)

### ISSUE THREAD

**Issue by dsbaars** [openstate/npo-backstage-examples#4]
I'm missing geo-related filters like [geo_bounding_box](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geo-bounding-box-filter.html) and [geo_distance_range](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geo-distance-range-filter.html). 

Example for NPO Journalistiek:

``` JSON
 {
    "filters": {
        "geo_bounding_box": {
            "Locations. GeoPoint": {
                "top_left": [52.1181, 5.7777],
                "bottom_right": [52.8542, 7.0728]
            }
        }
    }
}
```

**Comment by siccovansas** [comment]
Good point. There are currently just 408 items with locations (only in the `journalistiek` index), so at this moment we will not be adding this functionality. If geo-tagged items become more prominent in the future then we will reconsider this.

For now, we added a new function today to check if a field exists/contains a value (by posting an empty list to a filter of a field). You can use this to easily retrieve all items with locations:

```
curl -XPOST 'http://backstage-api.npo.nl/v0/journalistiek/search' -d '{"filters": {"Locations": []}}'
```
